common-files/Jenkinsfile

```diff
@@ -6,5 +6,6 @@ buildPlugin(
   useContainerAgent: true,
   configurations: [
     [platform: 'linux', jdk: 11], // use 'docker' if you have containerized tests
-    [platform: 'windows', jdk: 11]
+    [platform: 'windows', jdk: 11],
+    [platform: 'linux', jdk: 17]
```

Comment:
For most plugins I would rather recommend just Linux/17 + Windows/11 so we do not waste CI time, under the assumption that it will be rare for there to be a problem which appears on Java 11 but not 17 _and_ appears on Linux but not Windows.

Reply:
You can have your preference and I can have mine. I like having the Linux Java 11 configuration present because it is the common case for end users and it is fast, while Windows configurations are slow.